R/sensorreading-data.R

```diff
@@ -63,16 +67,29 @@ formatSensorData <- function(readings, columnNames, includeComments){
   
   for(listRows in row.names(readings)){
     listElements <- readings[listRows,]
-    
-    timeFormatted <- timeFormatting(listElements[["displayTime"]], "%m/%d/%Y")
-    timeFormattedCorrected <- timeFormatting(listElements[["nearestCorrectedTime"]], "%m/%d/%Y")
+		
+    if (!isEmptyOrBlank(listElements[["displayTime"]])){
+    	displayTime <- flexibleTimeParse(listElements[["displayTime"]], timezone, FALSE, TRUE)
+    }
+    if (!isEmptyOrBlank(listElements[["nearestCorrectedTime"]])){
+    	nearestCorrectedTime <- flexibleTimeParse(listElements[["nearestCorrectedTime"]], timezone, FALSE, TRUE)
+    }
+    timeFormatted <- timeFormatting(displayTime, "%m/%d/%Y", " ")
+    timeFormattedCorrected <- timeFormatting(nearestCorrectedTime, "%m/%d/%Y", " ")
 
     rec <- getRecorderWithinUncertainty(listElements[["uncertainty"]], listElements[["value"]], listElements[["recorderValue"]])
     ind <- getIndicatedCorrection(listElements[["recorderValue"]], listElements[["value"]])
     app <- getAppliedCorrection(listElements[["nearestRawValue"]], listElements[["nearestCorrectedValue"]])
     corr <- getCorrectedRef(listElements[["value"]], listElements[["nearestCorrectedValue"]], listElements[["uncertainty"]])
     
-    qual <- formatQualifiersStringList(as.data.frame(listElements[["qualifiers"]]))
+    qualifiers <- tryCatch({
```

Comment:
I think we've been putting try/catch blocks like this into `utils-parse.r` as "parse" functions. So this could be `parseSRSQualifiers`